enterprise_access/apps/subsidy_access_policy/models.py

```diff
@@ -1060,6 +1076,28 @@ def spend_available(self):
         # self.total_allocated is NEGATIVE USD Cents representing currently allocated assignments.
         return max(0, super().spend_available + self.total_allocated)
 
+    @memoize_instance_method
+    def get_assignment(self, lms_user_id, content_key):
+        """
+        Helper to get a ``LearnerContentAssignment`` for the given learner/content identifier pair
+        in this policy's assignment configuration.  Returns None if no such pair is assigned.
+        """
+        return assignments_api.get_assignment_for_learner(
+            self.assignment_configuration,
+            lms_user_id,
+            content_key,
+        )
+
+    def get_list_price(self, lms_user_id, content_key):
+        """
+        Uses the relevant assignment for this policy instance to determine the appropriate
+        list price of the requested ``content_key``.
+        """
+        found_assignment = self.get_assignment(lms_user_id, content_key)
+        if not found_assignment:
+            return super().get_list_price(lms_user_id, content_key)
+        return list_price_dict_from_usd_cents(found_assignment.content_quantity * -1)
```

Comment:
Please document the expected sign of found_assignment.content_quantity, and the desired sign of get_list_price return value in the docstring.  Thx!